ui/app/src/components/dialogs/CreateDashboardDialog.tsx

```diff
@@ -26,7 +26,7 @@ import {
 
 interface CreateDashboardProps {
   open: boolean;
-  projectOptions: string[];
+  projectOptions: ProjectResource[];
```

Comment:
Ok to have direct projects as props, but you could rename it to projects and build a list from projects like 
```ts
const projectOptions = projects.map((p) => {
  return {name: p.metadata.name, displayName: getResourceDisplayName(p)}
})
```

That would simplify then the rest of the code.

Reply:
I can rename to `projects` but I don't see the advantage to build `projectOptions`